components/mpas-seaice/bld/build-namelist

```diff
@@ -620,6 +620,7 @@ add_default($nl, 'config_recover_tracer_means_check');
 # Namelist group: column_package #
 ##################################
 
+add_default($nl, 'config_column_physics_type');
```

Comment:
Here and elsewhere for this namelist option, I'm not sure if under the `column_package` group is the right place for it, but I kept it consistent with the existing entry in `Registry.xml`.